src/components/Settings/TagsSettings.vue

```diff
@@ -216,6 +217,15 @@ export default {
       this.tagTextColor = '#f4b41a';
 
       this.$forceUpdate();
+      this.focusInput();
+    },
+
+    focusInput() {
+      setTimeout(() => {
+        if (this.$refs.tagInput) {
```

Comment:
I think it's safe to remove this check as well since 100ms is plenty of time for input to render

Reply:
I added this check to avoid an error when loading the general settings modal. It works inside the tags settings modal, but not earlier.